src/dev/MAX31855.cpp

```diff
@@ -0,0 +1,25 @@
+#include <EVT/io/SPI.hpp>
+#include <dev/MAX31855.h>
+
+using namespace EVT::core::IO;
+
+namespace DEV {
+
+MAX31855::MAX31855(SPI& spi, uint8_t dev) : spi(spi) {
+    this->device = dev;
```

Comment:
This can be done more safely in the initializer list instead of in the constructor body